ci-operator/config/redhat-openshift-ecosystem/openshift-preflight/redhat-openshift-ecosystem-openshift-preflight-main__4.8.yaml

```diff
@@ -11,8 +11,7 @@ images:
     FROM registry.access.redhat.com/ubi8/ubi:latest
 
     RUN cat /etc/os-release
-    ARG OPERATOR_SDK_VERSION=1.22.2
-    ARG OPENSHIFT_CLIENT_VERSION=4.7.19
```

Comment:
would this default to latest if not set ?

Reply:
nvm I see line 32

Reply:
Correct, we won't pin to a version anymore we'll just get latest. Which I think is fine for this.

Reply:
Closing this comment since we are removing `oc` dependency.